decidim-core/app/cells/decidim/follow_button_cell.rb

```diff
@@ -15,6 +15,14 @@ def show
 
     private
 
+    def followers_count
+      if model.respond_to?(:followers_count)
```

Comment:
I think we could even save this into an instance variable so that we skip the cached query (in case `model.followers.count` gets called)